src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -17,8 +17,13 @@
 
 import java.io.*;
 import java.util.*;
+import java.util.concurrent.atomic.*;
+
 import org.jitsi.util.*;
 import org.jitsi.videobridge.rest.*;
+import org.json.simple.*;
+
+import static org.jitsi.videobridge.EndpointMessageBuilder.createSelectedUpdateMessage;
```

Comment:
The org.json.simple package seems unused, use * for the static import.

Reply:
fixed